CHANGELOG.md

```diff
@@ -4,7 +4,13 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
-* Update Razor to 9.0.0-preview.24327.6 (PR: [#7273](https://github.com/dotnet/vscode-csharp/pull/7273))
+* Update Razor to 9.0.0-preview.24360.1 (PR: [#7327](https://github.com/dotnet/vscode-csharp/pull/7327))
+  * Improve perf in generator cache cases (PR: [#10577](https://ryzngard@github.com/dotnet/razor/pull/10577))
+  * Handle InsertReplaceEdit for completion (PR: [#10563](https://ryzngard@github.com/dotnet/razor/pull/10563))
+  * Use System.Text.Json for serialization (PR: [#10551](https://ryzngard@github.com/dotnet/razor/pull/10551))
+  * Support `DocumentSymbol` results from Roslyn (PR: [#10560](https://ryzngard@github.com/dotnet/razor/pull/10560))
+  * Merge release/dev17.11 to main (PR: [#10555](https://ryzngard@github.com/dotnet/razor/pull/10555))
+  * Fixing regression in legacy Razor editor used in .Net Framework projects (#10542) (PR: [#10554](https://ryzngard@github.com/dotnet/razor/pull/10554))
```

Comment:
Not off topic: All of these links have your username in them, which seems odd, though probably not damaging in any way.

100% off topic: having things in the release notes here that are unrelated to VS Code annoys me 😛

Reply:
I thought I got them all :( 

This is because it forces username auth for my URL... I'll fix